### ISSUE TITLE
lint/busySelect: false positive - listening on websocket connection

### ISSUE THREAD

**Issue by bontequero** [go-critic/go-critic#658]
I am using `gorilla/websocket` library and have code that looks something like this:
```go
for {
	select {
	case <-ctx.Done():
		return
	default:
		mt, wsmsg, err := conn.ReadMessage()
		if err != nil {
			log.Println(err)
			return
		}
		// Handle message
	}
}
```
Always listening on socket is desired behavior.

**Comment by Quasilyte** [comment]
This checker is not a part of the gocritic anymore.
